doc/tutorials/sklearn/sklearn_predict_abalone.ipynb

```diff
@@ -5,200 +5,71 @@
    "id": "a81062d6-1ec8-43f6-923a-65b8b73d7655",
    "metadata": {},
    "source": [
-    "In this notebook we will use a <a href=\"https://scikit-learn.org/stable/\" target=\"_blank\" rel=\"noopener\">`scikit-learn`</a> model created earlier to predict the age of an abalone from its physical measurements and sex. You can find more information about the problem domain <a href=\"https://archive.ics.uci.edu/dataset/1/abalone\" target=\"_blank\" rel=\"noopener\">here</a>.\n",
+    "# Predicting with Regression Model\n",
     "\n",
-    "We will be using a generic prediction UDF script. To execute queries and load data from Exasol database we will be using the <a href=\"https://github.com/exasol/pyexasol\" target=\"_blank\" rel=\"noopener\">`pyexasol`</a> module.\n",
+    "In this notebook, we will use a <a href=\"https://scikit-learn.org/stable/\" target=\"_blank\" rel=\"noopener\">`scikit-learn`</a> model created earlier to predict the age of an abalone from its physical measurements and sex. You can find more information about the problem domain <a href=\"https://archive.ics.uci.edu/dataset/1/abalone\" target=\"_blank\" rel=\"noopener\">here</a>.\n",
     "\n",
-    "Prior to running this notebook one needs to complete the follow steps:\n",
+    "We will be using a generic prediction UDF script. To execute queries and load data from the Exasol database we will be using the <a href=\"https://github.com/exasol/pyexasol\" target=\"_blank\" rel=\"noopener\">`pyexasol`</a> module.\n",
+    "\n",
+    "## Prerequisites\n",
+    "\n",
+    "Prior to using this notebook the following steps need to be completed:\n",
     "1. [Create generic scikit-learn prediction UDF script](sklearn_predict_udf.ipynb).\n",
-    "2. [Train a model on the Abalone](sklearn_train_abalone.ipynb)."
+    "2. [Train a model on the Abalone dataset](sklearn_train_abalone.ipynb).\n",
+    "\n",
+    "## Setup\n",
+    "\n",
+    "### Access configuration"
```

Comment:
We are not actually configuring anything here. We are picking up the secret store. The name "Access configuration" is probably misleading. If it confuses you it will confuse other people. I am trying to think of a better one. Suggestions are very welcome.

Reply:
This header is the same in every notebook, not just in this lot. Maybe let's leave it as it is for now. When we decide on a better name I will change it everywhere.